test/ProgressOnderwijsUtils.Tests/ParameterizedSqlTest.cs

```diff
@@ -156,7 +156,7 @@ public void ParameterizedSqlValidation()
             // ReSharper disable once NotAccessedVariable
             ParameterizedSql ignore;
             // ReSharper disable once AssignNullToNotNullAttribute
-            Assert.Throws<ArgumentNullException>(() => ignore = ParameterizedSql.CreateDynamic(null));
+            Assert.Throws<ArgumentNullException>(() => ignore = ParameterizedSql.CreateDynamic(null!));

```

Comment:
heeft deze assert nog zin dan?

Reply:
Het disabled de warning, aangezien null eigenlijkt niet toegestaan is. Aangezien dit een test is lijkt me dat wel acceptabel.

Reply:
En is het dan nog zinvol om te checken op arg null, in de method under test? Ik zou verwachten dat dat overbodig wordt